feat: Add benefit section component

jsandinoDev opened the pull request:
<!-- Do not delete this PR template. Just edit it to include the required information -->

# Description

<!-- If your PR fixes an open issue, use Closes #999 to link your PR with the issue. #999 stands for the issue number you are fixing -->

<!-- Github Issue Example: Closes #31 -->

Closes #9

# Changes proposed

## What were you told to do?
Build the Benefits Section Component 

<!-- Write the title of the issue/feature you are working on -->

## What did you do?
- Add Benefit Component in app/components
- Use this component in app/page

<!-- Talk about the things you did eg. files changes, dependencies installed e.t.c -->

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](../CONTRIBUTING.md) for this repository.

<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->

<!--
[x] - Correct; marked as done
[X] - Correct; marked as done
[ ] - Correct; marked as *not* done

[] - Not Correct; syntax error
[ x] - Not Correct; space between the brackets
-->

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the *main branch* (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/Videos
![Screenshot 2025-02-23 at 10 31 49 PM](https://github.com/user-attachments/assets/c45e9f1c-c5e8-4287-9080-ce5b19ccf40e)
![Screenshot 2025-02-23 at 10 32 14 PM](https://github.com/user-attachments/assets/9f259d7a-cca4-4829-bb70-82e59bdc603a)

<!-- If the changes are static page changes or UI changes add screenshots -->
<!-- If the changes involve implementing a functionality or working with apis, include a video
detailing how to implement the functionality and the request to the api and responses from the api endpoint-->
<!-- Add all the screenshots/videos which support your changes i.e before your change and after your change -->

Nwanne-san commented:
can you make this component responsive ? @jsandinoDev

jsandinoDev commented:
Fixed Responsive

<img width="837" alt="Screenshot 2025-02-24 at 6 42 18 PM" src="https://github.com/user-attachments/assets/42782942-890b-436a-9ffa-ab5dbb7f482f" />



There is a general problem in the responsive un mobile:
This problem is in the MainPage, I recommend yo create a new issue to fix this

<img width="616" alt="Screenshot 2025-02-24 at 6 42 57 PM" src="https://github.com/user-attachments/assets/4a0cc833-3a71-4353-9b05-647a647f78b0" />
<img width="580" alt="Screenshot 2025-02-24 at 6 44 04 PM" src="https://github.com/user-attachments/assets/0857cd4b-1d20-403f-8728-7d8f30511638" />

CMI-James commented:
Great work! Just a slight correction:  

- Adjust the **Why Choose StreamFi** components to use `col-span-3` and `col-span-2` instead of `col-span-2` and `col-span-1`.  
- Also, I noticed the images for each component are missing. Could you confirm if they were left out intentionally or if they need to be added?  

Let me know once the changes are made. Thanks

jsandinoDev commented:
Hi @CMI-James  Why change the col spans?

I just changed and its affect the location of the Cards:

(It is zoom out just to explain my point)
![Screenshot 2025-02-25 at 7 33 01 PM](https://github.com/user-attachments/assets/8da21686-2d92-46ef-84a4-b99e877825ad)


And for the images: I could not find them so I just fixed the background
